application/core/Survey_Common_Action.php

```diff
@@ -850,23 +850,19 @@ function _surveybar($aData)
 
             $aData['gid'] = $gid; // = $this->input->post('gid');
 
-            if (Permission::model()->hasSurveyPermission($iSurveyID, 'surveycontent', 'read'))
-            {
+            if (Permission::model()->hasSurveyPermission($iSurveyID, 'surveycontent', 'read')) {
```

Comment:
@olleharstedt See https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md - which says that in control structures the { on the same line is fine. Next line brackets applie only to functions/methods and classes.